roles/openshift_control_plane/handlers/main.yml

```diff
@@ -10,6 +10,13 @@
   until: result.rc == 0
   notify: verify API server
 
+- name: restart master controllers
+  command: "systemctl restart {{ openshift_service_type }}-master-controllers"
+  retries: 5
+  delay: 5
+  register: result
+  until: result.rc == 0
```

Comment:
doesn't seem related to delivering unit files

Reply:
But we don't call this handler from any where. You want this to be deleted as well?